modules/rostests/kmtests/tcpip/connect.c

```diff
@@ -166,6 +166,7 @@ TestTcpConnect(void)
             FALSE,
             NULL);
         Status = Irp->IoStatus.Status;
+        trace("Associate address IRP completed.\n");
```

Comment:
What's the purpose of that trace? I don't see its point, nor why it's added in this PR.
Can you remove it?

Reply:
Removed.